docs/Config_Reference.md

```diff
@@ -2471,6 +2471,24 @@ clock_pin:
 #initial_BLUE: 0.0
 #   See the "neopixel" section for information on these parameters.
 ```
+## [PCA9533]
+PCA9533 LED support. The PCA9533 is used on the mightyboard.
+
+```
+[pca9533 my_pca9533]
+#i2c_bus:
+#    The i2c peripheral, "twi" on the atmega
+#i2c_address: 98
+#    The i2c address, 98 or 99
```

Comment:
If there is a default, please state what the default is.  Otherwise state that "this parameter is required".  See the other config examples (eg, see https://www.klipper3d.org/Config_Reference.html#mcp4728 ).